src/sentry/static/sentry/app/components/sidebar/sidebarDropdown/index.jsx

```diff
@@ -33,6 +34,17 @@ class SidebarDropdown extends React.Component {
     onClick: () => {},
   };
 
+  handleLogout = (...args) => {
+    let {api} = this.props;
+
+    api.request('/auth/', {
+      method: 'DELETE',
```

Comment:
I'm guessing our client falls back to `POST` + `?_method=delete` or something here?

Reply:
The browser's XMLHttpRequest adapter can do DELETE requests. Very old versions of IE couldn't but those wouldn't be tall enough to ride a react app anyways.

Reply:
Yea, but the endpoint is a `POST` (just wondering where the translation happens).

Reply:
The /api/0/auth endpoint supports post, put and delete methods. The api endpoint was already built and not part of these changes. I also added a post method to the web endpoint so that the old url no longer does a logout on get.

Reply:
Ahhhh, whoops. Thought the endpoint you were changing here was this one.


![](https://media.giphy.com/media/4pMX5rJ4PYAEM/giphy.gif)